include/RAJA/policy/tbb/reduce.hpp

```diff
@@ -74,11 +75,18 @@ namespace RAJA
 
 namespace detail
 {
-template <typename T, typename Op>
+template <typename T, typename Reduce>
 class ReduceTBB
 {
 public:
-  using Reduce = Op;
+  struct reduce_adapter {
+    T operator()(T const &l, T const &r)
+    {
+      T tmp = l;
+      Reduce{}(tmp, r);
+      return tmp;
+    }
+  };
   using value_type = T;
```

Comment:
For consistency, using directives like this should probably be placed before other type definitions, methods, etc.